src/storage.rs

```diff
@@ -118,6 +118,12 @@ where
     /// Sets the latest accepted round.
     fn set_accepted_round(&mut self, na: R);
 
+    /// Sets a vector of max promises suffix.
+    fn set_max_promise_sfx(&mut self, max_promise_sfx: Vec<Entry<R>>);
+
+    /// Returns a vector of max promises suffix.
```

Comment:
```suggestion
    /// Returns the stored suffix of the maximum promise. Since this is only used once by the leader in the Prepare phase, it is recommended to return the consumed value.
```